src/types.ts

```diff
@@ -35,29 +36,37 @@ export type PageTreeItem = RawPageMetadataWithPublishedState & {
   path: string;
 };
 
+export type GlobalOptions = {
+  fieldsGroupName?: string;
+  slugSource?: SlugOptions['source'];
+  slugValidationRules?: ValidationBuilder<SlugRule, SlugValue>
+}
+
```

Comment:
Now that we expose the Options outside of the definePageType function / file, a more specific name would be better, like PageTypeOptions.

By defining in the page tree config, global is implied.